modules/common/tls/tls.go

```diff
@@ -125,3 +126,27 @@ func (t *TLS) CreateVolumes() []corev1.Volume {
 
 	return volumes
 }
+
+// CreateDatabaseClientConfig - connection flags for the MySQL client
+// Configures TLS connections for clients that use TLS certificates
+// returns a string of mysql config statements
+func (t *TLS) CreateDatabaseClientConfig() string {
+	conn := []string{}
+	// This assumes certificates are always injected in
+	// a common directory for all services
+	if t.Service.SecretName != "" {
+		conn = append(conn,
+			"ssl-cert=/etc/pki/tls/certs/tls.crt",
```

Comment:
is this for when client certs are used?

Reply:
correct, it might be useful in the future if we enable client cert-verification in mariadb.
The idea is that this name is generic, and the real certificate for the service is copied by kolla_start into this generic location.
https://github.com/openstack-k8s-operators/keystone-operator/blob/1e32653ddc594eda8428fc0335608b7befbb1a14/templates/keystoneapi/config/keystone-api-config.json#L35

Reply:
We might want to use a similar copy mechanism in kolla_start, and agree on a cert name to use here, rather than updating all service's kolla config to copy them manually?

Reply:
yes, we could do that and have pre-defined names for client/server cert